src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -244,6 +248,8 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
             List<StudentAttributes> studentList = StudentsLogic.inst().getStudentsForCourse(courseId);
 
             for (StudentAttributes student : studentList) {
+                if(email.equalsIgnoreCase(student.email) && isExculdingSelf)
+                    break;
```

Comment:
Shouldn't this be `continue` instead of `break`?

Reply:
Oh damn yeah. Thanks for pointing out.